aws/lambda/python/reports_notification.py

```
@@ -37,31 +40,54 @@ def lambda_handler(event, context):
     github_commit_sha = data['github']['sha']
 
     codebuild_logs_link = f"<{codebuild_link}|CodeBuild Logs>"
-    github_commit_link=f"<{gh_link}|{github_commit_sha[:7]}>"
+    github_commit_link = f"<{gh_link}|{github_commit_sha[:7]}>"
 
     build_succeeding = data['build_succeeding']
 
     if build_succeeding == "1":
         return
```

Ruff

Do not implicitly `return None` in function able to return non-`None` value

Add explicit `None` return value

(RET502)